plugins/outputs/syslog/framing.go

```diff
@@ -0,0 +1,64 @@
+package syslog
```

Comment:
This seems a copy of https://github.com/influxdata/telegraf/blob/master/plugins/inputs/syslog/framing.go.

I personally try to avoid code duplication. So probably would be better to use the `"input/syslog/Framing"` type directly.

And BTW if we do not want plugins belonging to different categorization (inputs vs outputs) to depend on each other pieces we can factor our common pieces.

WDYT @danielnelson ?

Reply:
I hesitated to extract it from the input plugin and include it somewhere within telegraf/internal/ but I didn't see anything plugin specific in there, so I went with the copy&paste (I'm not a fan either :) )

Reply:
Let's wait for Daniel :)

P.S.: needs some time to do a squash of the commit github has done for me :')